scripts/create_ipyparallel_mpi_profile.sh

```diff
@@ -6,11 +6,6 @@ IPYTHONDIR=~/.ipython
 # Create a new profile, called "mpi"
 ipython profile create --parallel --profile=mpi
 
-# Add the following line as per instructions from
```

Comment:
out of curiosity, why are you dropping this? unnecessary?

Reply:
Yes, the machinery in line 12 has been deprecated.
The option engines=mpi I am using here:

https://github.com/devitocodes/devito/pull/1923/files#diff-4acc409654fcff44dde5c96e156d25a86f29d10a612d110f245af6f76a47a443R25

is taking care of instructing the MPI launchers.